package.json

```diff
@@ -134,7 +143,7 @@
     "express-session": "^1.16.2",
     "express-validator": "^5.3.1",
     "file-type": "^16.0.1",
-    "firefox-profile": "^1.2.0",
+    "firefox-profile": "^4.0.0",
```

Comment:
This is only used in `openh264.js` and at this point i'd rather get rid of it all of the openH264 tooling